go.mod

```diff
@@ -26,6 +26,7 @@ require (
 	github.com/package-url/packageurl-go v0.1.3
 	github.com/pandatix/go-cvss v0.6.2
 	github.com/spdx/tools-golang v0.5.4
+	github.com/stretchr/testify v1.9.0
```

Comment:
These 3 new dependencies seem to be only used for one assert, can it be done manually instead?